--- ./tux.sql	2005-07-18 21:17:25.000000000 +0300
+++ /tmp/tux.sql	2005-07-18 22:12:37.000000000 +0300
@@ -36,13 +36,13 @@
   `id` bigint(20) unsigned NOT NULL auto_increment,
   PRIMARY KEY  (`id`),
   KEY `term` (`term`)
-) TYPE=MyISAM
+) TYPE=MyISAM;
 
 --
 -- table structure for table 'users'
 --
 
- CREATE TABLE `users` (
+CREATE TABLE `users` (
   `nick` char(50) NOT NULL default '',
   `timestamp` timestamp(14) NOT NULL,
   `status` tinyint(4) NOT NULL default '0',
